DEVELOPER_GUIDE.md

```diff
@@ -53,10 +54,43 @@ Bound addresses will then be logged to the terminal :
 [main] INFO  transportservice.TransportService - profile [test]: publish_address {127.0.0.1:5555}, bound_addresses {[::1]:5555}, {127.0.0.1:5555}
 ```
 
+
+
+## Create extensions.yml file
+
+Every extension will require metadata stored in an extensions.yml file in order to be loaded successfully.  In order to make the SDK look like an extension within OpenSearch, there must be an entry for the SDK within `extensions.yml`.
+
+- Start a separate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`.
+- Run `./gradlew assemble` to create a local distribution.
+- Navigate to the project root directory (i.e. `cd distribution/archives/linux-tar/build/install/opensearch-3.0.0-SNAPSHOT/`). Note: On Mac OS `linux-tar` should be replaced with `darwin-tar`.
+
+- Check if extensions directory exists in OpenSearch using `ls`.
+- If the directory does not exist, create it using `mkdir extensions`.
+- Navigate to the extensions folder using `cd extensions`.
+- Manually create a file titled `extensions.yml` within the extensions directory using an IDE or an in-line text editor.
+
+Sample extensions.yml file:
+
+```
```

Comment:
Syntax highlighting for the win!  Add `yaml` or `yml` here.

Reply:
Could you clarify where and how I should add that?